examples/cms-batch/gen.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+import itertools
+import random
+import sys
+
+FEATURES = ["few", "small", "equal", "max"]
+
+
+def gen(input_name: str) -> None:
+    max_n = [10, 10**5]["few" not in input_name]
+    max_k = [10**5, 10**9]["small" not in input_name]
+
+    if "max" in input_name:
+        n = max_n
+        k = max_k
+    else:
+        n = random.randint(1, max_n)
+        k = random.randint(n, max_k)
+
+    if "equal" in input_name:
+        k = n
+
+    print(n, k)
+
+
+# No arguments - List all inputs we can generate
+if len(sys.argv) == 1:
+    for variant in itertools.product([True, False], repeat=4):
+        features = [f for i, f in enumerate(FEATURES) if variant[i]]
+        if "equal" in features and "small" not in features:
+            continue  # Any equal input is small
+
+        if len(features):
```

Comment:
Empty collections ale falsy.

```suggestion
        if features:
```